### PR TITLE
feat: merge 'poe dev' and 'poe serve' as 'poe api --dev', rename 'poe…

### PR DESCRIPTION
**Pull request by RubenPants**
This pull request will:

- Merge `poe dev` and `poe serve` into `poe api [--dev]`
- Rename `poe streamlit` to `poe app`

### CODE REVIEW

File: {{ cookiecutter.__package_name_kebab_case }}/pyproject.toml [hunk 1]
```diff
@@ -169,17 +169,70 @@ target-version = "py{{ cookiecutter.python_version|replace('.', '') }}"
...
+  [tool.poe.tasks.api]
+  help = "Run a REST API"
+  shell = """
+    if [ $dev ]
+    then {
+      uvicorn \
+        --host $host \
+        --port $port \
+        --reload \
+        minifigures_api.api:app
```
Comment (lsorber) [inline {{ cookiecutter.__package_name_kebab_case }}/pyproject.toml:181]:
```suggestion
        {{ cookiecutter.__package_name_snake_case }}.api:app
```

File: {{ cookiecutter.__package_name_kebab_case }}/pyproject.toml [hunk 2]
```diff
@@ -169,17 +169,70 @@ target-version = "py{{ cookiecutter.python_version|replace('.', '') }}"
...
+      --access-logfile - \
+      --bind $host:$port \
+      --graceful-timeout 10 \
+      --keep-alive 10 \
+      --log-file - \
+      --timeout 30 \
+      --worker-class uvicorn.workers.UvicornWorker \
+      --worker-tmp-dir /dev/shm \
+      --workers 2 \
+      minifigures_api.api:app
```
Comment (lsorber) [inline {{ cookiecutter.__package_name_kebab_case }}/pyproject.toml:193]:
```suggestion
      {{ cookiecutter.__package_name_snake_case }}.api:app
```

**Review by lsorber — CHANGES_REQUESTED** [review]
LGTM! Just a few small suggestions.

Did you try the `--dev` flag? I didn't scaffold a project with this branch to test.

File: {{ cookiecutter.__package_name_kebab_case }}/pyproject.toml (hunk 1 above)
Comment (RubenPants) [reply {{ cookiecutter.__package_name_kebab_case }}/pyproject.toml:181]: Yes I did try out the flag. 
Thank you for fixing the project name 😓

**Review by lsorber — APPROVED** [review]
LGTM, thanks for the improvement!

**Pull request merged**